pkg/build_tar.py

```diff
@@ -90,6 +90,16 @@
 FLAGS = flags.FLAGS
 
 
+def GetFlagValue(flagvalue, strip=True):
```

Comment:
If  you are going to bring this in, please take the version from make_deb.py, which is PY2 and PY3 compatible. https://github.com/bazelbuild/rules_pkg/blob/96e34abe84353387fd378fba48be88836f6c3f2d/pkg/make_deb.py#L310

Reply:
Maybe extracting this into a library first is the right approach.  The helpers already depend on another library (archive), so structurally it is just another file and an import.